apps/emqx/src/emqx_alarm.erl

```diff
@@ -571,3 +589,13 @@ worker_loop() ->
             end,
             ?MODULE:worker_loop()
     end.
+
+maybe_monitor_process_related_alarms(
+    <<"conn_congestion/", _/binary>> = Name,
+    Pid,
+    #{pmon := Pmon} = State
+) ->
+    Pmon1 = emqx_pmon:monitor(Pid, Name, Pmon),
```

Comment:
`emqx_pmon` is perhaps not a good fit for this, it implies that one process can have only one associated value (`Name` in this case). i.e. one process can only have one alarm.